DCSCN.py

```diff
@@ -77,6 +85,8 @@ def __init__(self, flags, model_name=""):
         self.psnr_calc_border_size = flags.psnr_calc_border_size
         if self.psnr_calc_border_size < 0:
             self.psnr_calc_border_size = self.scale
+        # self.input_image_width = flags.input_image_width
```

Comment:
Hi, if it's commented, I think it's better to remove it. (same as other parts)
What is the purpose of this option? If it's useful, we can uncomment it since it's optional and wouldn't conflict with the other options.

Reply:
Hmm okay! I was questioning whether to put those flags in yet - the approach is to fix the input size of the model so that the frozen graph can be converted to tensorflow-lite, but I haven't tested the function with the updated code base yet.

Reply:
I see... In that case, I think the option is needed. Let me merge it first then we can add that option and test it later. Thx!